src/System.Runtime/tests/System/Text/StringBuilderTests.cs

```diff
@@ -979,6 +979,58 @@ public static void Clear()
             Assert.Same(string.Empty, builder.ToString());
         }
 
+        [Fact]
+        public static void Clear_Empty_CapacityNotZero()
+        {
+            var builder = new StringBuilder();
+            builder.Clear();
+            Assert.NotEqual(0, builder.Capacity);
+        }
+
+        [Theory]
+        [InlineData(14)]
```

Comment:
Why use a Theory with only one value? That would normally be a fact, or you could add more data.